src/fieldset/index.jsx

```diff
@@ -24,6 +25,14 @@ const fields = {
 };
 
 class Fieldset extends Component {
+
+  constructor(options) {
+    super(options);
+    this.state = {
+      model: {}
+    };
```

Comment:
rather than setting the state to be an empty model, could we just set it to this.props.model here - we can also do the binding in the constructor then you'd be able to remove the componentDidMount function